### ISSUE TITLE
Cannot find module 'lodash/fp' 

### ISSUE THREAD

**Issue by tommyboylab** [strapi/strapi#7246]
**Describe the bug**
Currently migration to Strapi 3.1.1 is failing because strapi / strapi-admin are using a broken lodash version [Issue tracked here](https://github.com/lodash/lodash/issues/4848)

**Steps to reproduce the behavior**

1. Update Strapi
2. Follow Migration Guide
3. Run Build -- Clean

**Expected behavior**
Lodash needs to be updated for migration to work properly

**Code snippets**
```[2020-07-29T10:44:26.269Z] debug ⛔️ Server wasn't able to start properly.
[2020-07-29T10:44:26.270Z] error Error: Cannot find module 'lodash/fp'
Require stack:
- /strapi/node_modules/strapi-admin/controllers/formatters/conditions.js
- /strapi/node_modules/strapi-admin/controllers/formatters/index.js
- /strapi/node_modules/strapi-admin/controllers/permission.js
- /strapi/node_modules/strapi/lib/load/load-files.js
-/strapi/node_modules/strapi/lib/core/load-apis.js
- /strapi/ode_modules/strapi/lib/core/load-modules.js
- /strapi/node_modules/strapi/lib/Strapi.js
- /strapi/node_modules/strapi/lib/index.js
- /strapi/node_modules/strapi/lib/commands/develop.js
- /strapi/node_modules/strapi/bin/strapi.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:966:15)
    at Function.Module._load (internal/modules/cjs/loader.js:842:27)
    at Module.require (internal/modules/cjs/loader.js:1026:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (/strapi/node_modules/strapi-admin/controllers/formatters/conditions.js:3:23)
    at Module._compile (internal/modules/cjs/loader.js:1138:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1158:10)
    at Module.load (internal/modules/cjs/loader.js:986:32)
    at Function.Module._load (internal/modules/cjs/loader.js:879:14)
    at Module.require (internal/modules/cjs/loader.js:1026:19)
```

**System**

- Node.js version: 12
- NPM version: 6.14.5
- Strapi version: 3.1.1
- Database: sqlite
- Operating system: Linux

**Comment by alexandrebodin** [comment]
Nice catch we just merged an update and we should release today

**Comment by alexandrebodin** [comment]
Just released 3.1.2 :) Closing
